backend/controllers/LocationsController.ts

```diff
@@ -180,15 +160,15 @@ const LocationsController = {
 
         const userLocation = new Location({
           locationName: locationName,
-          coordinates: coordinates,
+          coordinates: [+coordinates[0], +coordinates[1]],
           arrayPhotos: imageUrls,
           description: description,
           comments: [],
           rating: {
             likes: [],
             dislikes: []
           },
-          filters: [],
+          filters: filters.split(','),
           author: _id
```

Comment:
have you tested the case with empty filters?

and what do you receive from frontend when you don't set any filter?

Reply:
later